culture-n-values.md

```diff
@@ -52,13 +52,13 @@ This sounds good in theory, but __none__ of those parts happen as smoothly as th
 
-When Vinta talk about no estimates, we charge and agree on contracts based on hours worked, not features delivered. That means a project is never finished until the client is satisfied. Although scary at first, there are a lot of benefits that come from it:
+When Vinta talks about no estimates, we charge and agree on contracts based on hours worked, not features delivered. That means a project is never finished until the client is satisfied. Although scary at first, there are a lot of benefits that come from it:
 - **We don’t need to estimate**, since we saw it’s difficult and error prone, and that’s a big relief.
 - **Everything takes the time it should take to be done**, that means more testing, more studying and more time invested in quality, which leads to better code and increased maintainability.
-- **The client doesn’t need to waste time thinking/predicting every single need she/he might have**. Also, she/he is forced to stay engaged to the project and receives more deliveries according to what she/he wanted, instead of waiting a long period of time to see something that isn’t exactly right.
+- **The client doesn’t need to waste time thinking/predicting every single need she/he might have**. Also, she/he is forced to stay engaged to the project and receives more deliveries according to what she/he wanted, instead of waiting for a long period of time to see something that isn’t exactly right.
 
 # Why we believe in open source
-We are community people, open source contributions are wonderful  and everyone is encouraged to do some. Since a lot of our coding uses open-source tools, maintaining our own open-source projects seems reasonable. So we encourage people to contribute as projects demands. We find that is one way to leave our company's mark in the community as well is to always fork using [Vinta's Github account](https://github.com/vintasoftware).
+We are community people, open source contributions are wonderful and everyone is encouraged to do some. Since a lot of our coding uses open-source tools, maintaining our own open-source projects seems reasonable. So we encourage people to contribute as projects demands. We find that is one way to leave our company's mark in the community as well is to always fork using [Vinta's Github account](https://github.com/vintasoftware).
```

Comment:
A period would be better than a comma here:
```suggestion
We are community people. Open source contributions are wonderful and everyone is encouraged to do some. Since a lot of our coding uses open-source tools, maintaining our own open-source projects seems reasonable. So we encourage people to contribute as projects demands. We find that is one way to leave our company's mark in the community as well is to always fork using [Vinta's Github account](https://github.com/vintasoftware).
```